controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -195,6 +196,13 @@
                                          null);
     }
 
+    public CompletableFuture<ScaleStatusResponse> checkScale(final String scope, final String stream, final int epoch) {
```

Comment:
i have made the status an enum and if epoch is not found i am returning input_invalid (same if scope or stream is not found).